client/src/components/About.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import $ from 'jquery';
+
+const Paragraph = styled.p`
+color: #382110;
```

Comment:
not sure if this is just GitHub rendering strangely, but typical CSS practice is to indent styling. check out the top "hello world" example here: https://www.styled-components.com/docs/basics